node/src/reactor/main_reactor/keep_up.rs

```diff
@@ -293,13 +293,7 @@ impl MainReactor {
         // signatures against. we use the leaper to gain awareness of the necessary
         // trusted ancestors to our earliest contiguous block to do necessary validation.
         let leap_status = self.sync_leaper.leap_status();
-        info!(
-            ?parent_hash,
-            ?leap_status,
-            "Historical: {} {}",
-            parent_hash,
-            leap_status
-        );
+        info!("Historical status for {} is {}", parent_hash, leap_status);
```

Comment:
```suggestion
        info!("historical status for {} is {}", parent_hash, leap_status);
```

Reply:
Fixed in https://github.com/casper-network/casper-node/pull/3507/commits/8352a002fa1f86ed2a3ae82ec8440c3a00961e4e and https://github.com/casper-network/casper-node/pull/3507/commits/754a7d4d610ed5c667c11451813c7776c62d518b